spec/services/vendor_api/single_application_presenter_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe VendorApi::SingleApplicationPresenter do
+  subject(:presenter) { described_class.new(application_choice) }
+
+  let(:application_choice) { create :application_choice }
+
+  around do |example|
+    Timecop.freeze do
+      example.run
+    end
+  end
+
+  describe '#as_json' do
```

Comment:
Let's discuss this spec offline!

Reply:
I've reworked this spec to test the whole return value in one example and removed the `let`s.

I've not attempted to fill out all the fields that this presenter/serializer will eventually return, as they get added this test will need to be updated (it'll fail if anything at all changes so it should be fairly obvious what needs to be done!).